Sources/Alchemy/Database/QueryBuilder/Clauses/WhereClause.swift

```diff
@@ -68,7 +68,7 @@ public struct WhereIn {
 extension WhereIn: WhereClause {
     func toSQL() -> SQL {
         let placeholders = Array(repeating: "?", count: values.count)
-        return SQL("\(boolean) \(key) \(type)(\(placeholders))", bindings: values)
+        return SQL("\(boolean) \(key) \(type)(\(placeholders.joined(separator: ", ")))", bindings: values)
```

Comment:
To keep it cleaner, might be good to moved the joined part out of the string building step and into the line above